src/averageAges.js

```diff
@@ -37,7 +45,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenArray = people.filter(
+    (person, index, arr) => {
+      return person.sex === 'f'
```

Comment:
There is a repeating logic for sex checks. Consider using the function isMan or something similar, which would check the sex of the person and would return true/false values.